src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,51 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+
+        if (user.getLogin() == null) {
+            throw new InvalidUserDataException("User login can not be null!");
+        }
+
+        if (user.getPassword() == null) {
+            throw new InvalidUserDataException("User password can not be null!");
+        }
```

Comment:
Consider checking for empty strings in addition to null values for the password field. This will ensure that a user cannot register with an empty password.